src/tools/sdk/_windows.md

```diff
@@ -1,23 +1,41 @@
+###How to install?
```

Comment:
In this header, and others make sure to put a space after the final hash(`#`) or you'll end up with headers such as these:

![image](https://user-images.githubusercontent.com/18372958/128093098-f906d647-ba1c-4f32-a7e0-203da672cec4.png)

Also, I'm not sure we want to use headers within these OS specific instructions anyway, as then they will show up in the TOC even if a user does not have the Windows tab open. Then when clicking these with the Mac or Linux tab open, these TOC entries will not work(at least not properly). So perhaps we want to stick with the similar structure as before.

![image](https://user-images.githubusercontent.com/18372958/128093033-681ac558-aebd-4920-8c9d-e7b62496d510.png)

Reply:
Sure, will update.